SETUP.md

```diff
@@ -32,91 +32,65 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 1. Install Homebrew: `ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
 1. Install Redis: `brew install redis`
 1. Run `brew install https://raw.github.com/quantiverge/homebrew-binary/pdftk/pdftk.rb enscript gs mysql nvm imagemagick rbenv ruby-build coreutils sqlite phantomjs`
-    1. If it complains about `Formula.sha1` is disabled, removing https://raw.github.com/quantiverge/homebrew-binary/pdftk/pdftk.rb seems to not have serious side effects (it will cause `PDFMergerTest` to fail).
```

Comment:
Hmm? I didn't touch the OSX section at all. While trying to merge, my git kept complaining about merge conflicts with SETUP.md. To resolve it, I removed SETUP.md and replaced it with an identical version that had all the Ubuntu changes. Perhaps that's why it thinks things in the OSX section have been changed